mobile/src/modules/auth/screens/SendScreen.tsx

```diff
@@ -18,34 +16,64 @@ interface IProps extends InjectedReduxProps {
 }
 type Props = IProps & NavigationInjectedProps;
 
-const styles = StyleSheet.create({
-  hintView: {
-    paddingTop: Sizes.SMALL,
-    flexGrow: 1
-  }
-});
-
 class Send extends React.PureComponent<Props> {
   static navigationOptions = {
     headerTitle: <HeaderTitle>{i18n.t(core.screens.send.title)}</HeaderTitle>
   };
 
+  state = {
+    amountNQT: 0,
+    feeNQT: 0
+  };
+
   render () {
     return (
       <Screen>
         <FullHeightView>
-          <View style={styles.hintView}>
-            <Text theme={TextThemes.HEADER}>
-              {i18n.t(core.screens.send.title)}
-            </Text>
-          </View>
           <View>
-            <Text>Send Burst Form</Text>
+            <TextInput
+              style={{ height: 40 }}
+              keyboardType='email-address'
+              placeholder='BURST-____-____-____-_____'
+              textContentType='none'
+              onChangeText={this.handleBurstInputChange()}
+              autoCorrect={false}
+            />
+            <TextInput
+              style={{ height: 40 }}
+              keyboardType='decimal-pad'
+              textContentType='none'
+              placeholder={i18n.t(core.screens.send.amountNQT)}
+              onChangeText={this.handleInputChange('amountNQT')}
+              autoCorrect={false}
+            />
+            <TextInput
+              style={{ height: 40 }}
+              keyboardType='decimal-pad'
+              textContentType='none'
+              placeholder={i18n.t(core.screens.send.feeNQT)}
+              onChangeText={this.handleInputChange('feeNQT')}
+              autoCorrect={false}
+            />
+            <Button
+              title={i18n.t(core.actions.submit)}
+              onPress={this.handleSubmit}
+            />
           </View>
         </FullHeightView>
       </Screen>
     );
   }
+
+  private handleSubmit = () => {
+    // console.log(this.state.amountNQT, this.state.feeNQT);
+  }
+  private handleInputChange (key: string): ((text: string) => void) | undefined {
```

Comment:
There is no need to write private/public/etc, cuz we don't use (99%) inheritance between react components.
Also there is a convention(?) about methods order: static things, constructor, lifecycle methods, custom methods, render

Btw, this way of changing state is nice as shorthand, but we don't have proper TS-checks, because key is just a string. So if prefer use different methods to change different parts of state. Anyway, here we can use `key: [keyof State]`, and describe a State interface somewhere above.

Also here is types mistake: we have numbers in state, but passing string instead